identity-service/src/routes/wormhole.js

```diff
@@ -43,7 +43,7 @@ const relayWormhole = async (
     logs.push(`Attempting Transfer Tokens for sender: ${senderAddress}`)
     const { sha: transferTokensSHA, txProps: transferTokensTxProps } = getTxProps(senderAddress, transferTokens)
     const estimatedGas = await ethTxRelay.estimateEthTransactionGas(senderAddress, transferTokensTxProps.contractAddress, transferTokensTxProps.encodedABI)
-    const gasMultiplier = 1.5
+    const gasMultiplier = 1.05
     transferTokensTxProps.gasLimit = Math.floor(estimatedGas * gasMultiplier)
```

Comment:
can we not rely on the libs gas estimator?

Reply:
They estimate gas using different methods:
Libs relies on the contract being init and calls estimate gas on the method - https://github.com/AudiusProject/audius-protocol/blob/6c85d731ec80328d392101031039f4143dc3e770/libs/src/utils/estimateGas.js#L21-L25
In identity we just have the signed data - https://github.com/AudiusProject/audius-protocol/blob/6c85d731ec80328d392101031039f4143dc3e770/identity-service/src/relay/ethTxRelay.js#L100-L109